pkg/apis/core/types.go

```diff
@@ -4664,10 +4718,14 @@ type Secret struct {
 	Type SecretType
 }
 
+// MaxSecretSize represents the max secret size.
+// 1 * 1024 * 1024 = 1 MB
```

Comment:
we don't need the math in the godoc, can just say (1MB)

Reply:
This is done!